spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -90,12 +91,20 @@ private String clientName(String id) {
         return id;
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public ManagedChannelCustomizer managedChannelCustomizer() {
```

Comment:
This addition serves a simple additional verification in the `AxonServerAutoConfigurationTest` I think.

Reply:
Added test for this